frontend/app/component/form/polling_station/PollingStationForm.test.tsx

```diff
@@ -34,32 +34,122 @@ async function fillForm(user: UserEvent, testPollingStation: PollingStation | Om
   await userEvent.click(screen.getByRole("button", { name: "Opslaan en toevoegen" }));
 }
 
-describe("PollingStationForm create", () => {
-  test("PollingStationForm create", async () => {
-    const testPollingStation: Omit<PollingStation, "id"> = {
-      election_id: 1,
-      number: 1,
-      name: "test",
-      street: "test",
-      postal_code: "1234",
-      locality: "test",
-      polling_station_type: "FixedLocation",
-      number_of_voters: 1,
-      house_number: "test",
-    };
-
-    const onSaved = vi.fn();
-    render(<PollingStationForm electionId={1} onSaved={onSaved} />);
-
-    const user = userEvent.setup();
-    await fillForm(user, testPollingStation);
-
-    await userEvent.click(screen.getByRole("button", { name: "Opslaan en toevoegen" }));
+describe("PollingStationForm", () => {
+  describe("PollingStationForm create", () => {
+    test("Succesful create", async () => {
+      const testPollingStation: Omit<PollingStation, "id"> = {
+        election_id: 1,
+        number: 1,
+        name: "test",
+        street: "test",
+        postal_code: "1234",
+        locality: "test",
+        polling_station_type: "FixedLocation",
+        number_of_voters: 1,
+        house_number: "test",
+      };
+
+      const onSaved = vi.fn();
+      render(<PollingStationForm electionId={1} onSaved={onSaved} />);
+
+      const user = userEvent.setup();
+      await fillForm(user, testPollingStation);
+
+      await userEvent.click(screen.getByRole("button", { name: "Opslaan en toevoegen" }));
+
+      await waitFor(() => {
+        expect(onSaved).toHaveBeenCalledWith(expect.objectContaining(testPollingStation));
+      });
+    });
 
-    await waitFor(() => {
-      expect(onSaved).toHaveBeenCalledWith(expect.objectContaining(testPollingStation));
+    test("Validation", async () => {
```

Comment:
This test needs to be split up in three separate tests.

Reply:
i've rewritten this to separate tests